### PR TITLE
[DFFA-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by DevOge**

### CODE REVIEW

File: submissions/OgeEjim/DOM/signupval.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+//     if (field.value == '') {
+//     field.nextElementSibling.innerHTML = "Please enter your " + fieldName;      
+//     event.preventDefault();
+//     return false; 
+// } else{
+//     field.nextElementSibling.innerHTML = "";
+//     return true;
+// }
+// }
+
```
Comment (Ireti203) [inline submissions/OgeEjim/DOM/signupval.js:20]: Remove commented out code

File: submissions/OgeEjim/DOM/signupval.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+function valFl(firstName,event) {
```
Comment (Ireti203) [inline submissions/OgeEjim/DOM/signupval.js:21]: Kindly rename this function

File: submissions/OgeEjim/DOM/signupval.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        event.preventDefault();
+        return false;
+    } else {
+        lastName.nextElementSibling.innerHTML = "";
+           return true; 
+    } 
+}
+
+
+function valNum(number,event) {
```
Comment (Ireti203) [inline submissions/OgeEjim/DOM/signupval.js:55]: same here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/OgeEjim/DOM/signupval.js (hunk 3 above)
Comment (DevOge) [reply submissions/OgeEjim/DOM/signupval.js]: All done ma

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**